user_guide_src/source/libraries/uploaded_files/002.php

```diff
@@ -27,7 +27,10 @@ public function upload()
                 ],
             ],
         ];
-        if (! $this->validate($validationRule)) {
+
+        $postData = ['userfile' => $this->request->getPost('userfile')];
```

Comment:
In fact, in this case `$_POST` is an empty array. So `$postData` doesn't matter.
But is this better? Since it works even when other fields are present.

```suggestion
        $postData = $this->request->getPost(array_keys($validationRule));
```